plugins/chat/spec/system/page_objects/chat/chat_thread.rb

```diff
@@ -49,10 +49,25 @@ def click_send_message(id)
       end
 
       def has_message?(thread_id, text: nil, id: nil)
+        check_message_presence(thread_id, exists: true, text: text, id: id)
+      end
+
+      def has_no_message?(thread_id, text: nil, id: nil)
```

Comment:
I had to add these negative versions since if I used `expect(open_thread).not_to  have_message(thread.id, id: message_4.id)` or `expect(channel_page).not_to have_message(id: message_1.id)` it was waiting for the full 10 seconds each time, making the "when bulk deleting messages across the channel and a thread" spec take ~40s needlessly.